example/plugins/example/plugin.js

```diff
@@ -0,0 +1,17 @@
+class ExamplePlugin {
+  constructor(pod, config) {
+    this.pod = pod;
+    this.config = config;
+  }
+
+  onRendererCreate(renderer) {
+    renderer.env.addFilter('testPluginFilter', value => {
```

Comment:
Only the Nunjucks renderer has an `env` (or rather, we have no guarantee that other template engines also have an `env`, so I think this example would need to be modified to indicate or limit it somehow to Nunjucks.

Reply:
I was thinking that we could add a `kind` to the renderer, or something that would allow the plugin to know what renderer is in use. Then we could do this in the plugin:

```
if (renderer.kind === 'nunjucks') {
  ...
}
```

Reply:
I've added a `kind` property to renderers so that the plugin can target specific renderers in the trigger.